pkg/controllers/binding/common.go

```diff
@@ -193,6 +197,23 @@ func RecordAppliedOverrides(cops *overridemanager.AppliedOverrides, ops *overrid
 	return annotations, nil
 }
 
+// parseConflictResolution determine the conflictResolution annotation of Work: preferentially inherit from RT, then RB
+func parseConflictResolution(workload *unstructured.Unstructured, conflictResolutionInBinding policyv1alpha1.ConflictResolution,
+	annotations map[string]string) map[string]string {
+	conflictResolutionInWork := util.GetAnnotationValue(workload.GetAnnotations(), workv1alpha2.ResourceConflictResolutionAnnotation)
+
+	if conflictResolutionInWork != "" {
```

Comment:
`InWork` might be misleading, it seems like a `Work` object, but actually, it's a `resource template`.

In addition, please add a comment to explain why you can return in case of `conflictResolutionInWork != ""`.

Reply:
I fix this tip, and added some more modifications, plz re-read once more, thanks